upgrading/automated_upgrades.adoc

```diff
@@ -735,6 +735,71 @@ upgrade the component.
 To upgrade an existing EFK logging stack deployment, you review your parameters
 and run the *_openshift-logging/config.yml_* playbook.
 
+The EFK upgrade also upgrades Elasticsearch from the 2.x version to the 5.x version. For important information on changes in Elasticsearch 5.x, you should review the link:https://www.elastic.co/guide/en/elasticsearch/reference/5.5/breaking-changes-5.0.html[Elasticsearch breaking changes].
```

Comment:
I recommend changing from `[version].x` syntax in our documentation, even though I see that ES uses it in theirs - maybe try `version 5 and higher` or on line 758 `In Elasticsearch 5 the field...`